2019-11-23 15:42:22.219 - [Info] MITM Version: 1.26
2019-11-23 15:42:22.227 - [Info] Auto Access Enabled: true
2019-11-23 15:42:22.227 - [Debug] [Init] Auto Access Barrier: true
2019-11-23 15:42:22.251 - [Info] SSH man-in-the-middle server for 172.20.0.5 listening on 0.0.0.0:10104
2019-11-23 15:54:05.301 - [Debug] [Connection] Attacker connected: 193.188.22.229 | Client Identification: SSH-2.0-libssh2_1.7.0
2019-11-23 15:54:05.384 - [Debug] [Connection] Client error on ssh server
2019-11-23 15:54:05.386 - [Debug] [Connection] Attacker closed the connection
2019-11-23 16:40:06.534 - [Debug] [Connection] Attacker connected: 206.189.142.10 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-23 16:40:07.926 - [Debug] [Auth] Attacker 206.189.142.10 trying to authenticate with "password"
2019-11-23 16:40:07.929 - [Debug] [Auto Access] Attacker: 206.189.142.10, Threshold: 1, Attempts: 1
2019-11-23 16:40:07.929 - [Debug] [Auto Access] Compromising the honeypot
2019-11-23 16:40:07.929 - [Debug] [Auto Access] Adding the following credentials: "http:hhhhhhhhhh"
2019-11-23 16:40:08.402 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-23 16:40:08.402 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.5
2019-11-23 16:40:08.483 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-23 16:40:09.391 - [Debug] [Connection] Attacker connected: 206.189.142.10 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-23 16:40:10.784 - [Debug] [Auth] Attacker 206.189.142.10 trying to authenticate with "password"
2019-11-23 16:40:10.784 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-23 16:40:10.785 - [Debug] [Auth] Attacker: 206.189.142.10 has so far made 1 attempts. Remaining: 5 attempts
2019-11-23 16:40:11.260 - [Debug] [Connection] Attacker closed the connection
2019-11-23 16:40:11.689 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
